docs/install_kyma_dashboard_manually.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Install Kyma Dashboard manually
+---
+
+This document describes how to install Kyma Dashboard with Istio Ingress. It also provides instructions on how to install the Dashboard on a Kyma cluster.
+
+> **NOTE**: In both cases, you must export KUBECONFIG as an environment variable that points to the kubeconfig file you want to use.
+
+## Install Kyma Dashboard with Istio Ingress
+
+> **NOTE**: To install Kyma Dashboard manually with Istio Ingress, you must have Istio configured on your cluster. See [Istio Getting Started](https://istio.io/latest/docs/setup/getting-started/) for more information.
+
+1. Create a Namespace for Kyma Dashboard:
+
+`kubectl create namespace {NAMESPACE_NAME}`
```

Comment:
```suggestion
```kubectl create namespace {NAMESPACE_NAME}```
```

Reply:
Something I missed before - use code block when providing commands. I can't find a specific guideline now, but here's [the example](https://kyma-project.io/community/guidelines/content-guidelines/03-formatting/#omission-in-code).